.changeset/hot-friends-act.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/core-plugin-api': minor
+'@backstage/app-defaults': minor
+'@backstage/core-app-api': minor
+'@backstage/plugin-user-settings': patch
+---
+
+Integrate `auth-backend-module-openshift-provider` into the Backstage core and `@backstage/plugin-user-settings`.
```

Comment:
Could you split this into separate changesets that apply to each of these? Highlighting the additions to `core-plugin-api` vs the implementation in `app-defaults` and `core-app-api`, etc.